addons/sys_bf888s/functions/fnc_preset_information.sqf

```diff
@@ -0,0 +1,108 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Initialises and registers the default presets with the initial values of transmitting and receiving frequencies for each of the channels.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acre_sys_bf888s_fnc_preset_information
+ *
+ * Public: No
+ */
+
+private _presetFrequencies = [];
+for "_i" from 0 to 255 do {
```

Comment:
16 channels only it seems.

Reply:
Should this number be changed too to reflect 16 channels?

Reply:
Yeah 16 seems the channel limit. I see the preset `default` only has 16 channels, but `default2`, `default3` and `default4` all have more than 16 channels declared in this file (fnc_preset_information.sqf). I think making it so that all have presets have just 16 channels would be best.